### PR TITLE
Allow to give address when scanning.

### LINKED ISSUE #88: Support specifying address and port when scanning
``` sh
jag scan 192.168.0.27:9000
```

The device should respond with the same information it usually broadcasts over UDP, and should verify the Jaguar version but **not** the device id.

This is likely to have solved the issues reported in https://github.com/toitlang/jaguar/issues/87.

### PR DESCRIPTION
**Pull request by floitsch**
Fixes #88.

### CODE REVIEW

File: src/jaguar.toit [hunk 1]
```diff
@@ -167,8 +167,8 @@ install_firmware firmware_size/int reader/reader.Reader -> none:
     finally:
       writer.close
 
-broadcast_identity network/net.Interface id/uuid.Uuid name/string address/string -> none:
-  payload ::= json.encode {
+create_identification_payload id/uuid.Uuid name/string address/string -> ByteArray:
```
Comment (kasperl) [inline src/jaguar.toit:170]: identity_payload?

File: cmd/jag/commands/scan.go [hunk 2]
```diff
@@ -18,24 +21,32 @@ import (
...
-	scanPort    = 1990
+	scanTimeout  = 600 * time.Millisecond
+	scanPort     = 1990
+	scanHttpPort = 9000
 )
 
 func ScanCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "scan",
+		Use:   "scan [address]",
```
Comment (kasperl) [inline cmd/jag/commands/scan.go:31]: Maybe we can use this if the device option (-d) is an address?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: src/jaguar.toit (hunk 1 above)
Comment (floitsch) [reply src/jaguar.toit]: done.

File: cmd/jag/commands/scan.go (hunk 2 above)
Comment (floitsch) [reply cmd/jag/commands/scan.go:31]: Will do a follow-up PR.

Comment (floitsch) [reply cmd/jag/commands/scan.go:31]: https://github.com/toitlang/jaguar/pull/196

**Pull request merged**